### DISCUSSION TITLE
A research GitHub Action to check for Unsafe Dependency Changes

### DISCUSSION THREAD

**Discussion by supatsara-wat** [typicode/husky#1312] [Show and tell]
We would like to recommend an action we created to help Open Source Projects, especially when dealing with code changes that might be unsafe when updating dependencies.

FYI - This project was flagged as having such PRs.

https://github.com/marketplace/actions/depsafe.

The code is fairly simple, and it simply flags whenever a require() function is being introduced, as it brings in external resources.

Asia and Raula. @raux

**Comment by typicode** [comment]
Thank you for the suggestion. Currently, I'm fine with the way code changes are handled :)
